src/layers.ts

```diff
@@ -149,6 +149,6 @@ export const getLayerLangPaths = (nuxt: Nuxt) => {
     nuxt.options._layers
       .filter(layer => layer.config.i18n?.langDir != null)
       // @ts-ignore
-      .map(layer => pathe.resolve(layer.config.rootDir, layer.config.i18n.langDir)) as string[]
+      .map(layer => pathe.resolve(layer.config.rootDir, layer.config.srcDir, layer.config.i18n.langDir)) as string[]
```

Comment:
You're right, it looks like `srcDir` includes the absolute path, I thought it would use the configured value of `srcDir`. But as seen here `rootDir` is always part of `srcDir` on `_layers` https://github.com/nuxt/nuxt/blob/ae82d7089598bd1a4e5a8f88cd9edf7a1f12d7c7/packages/kit/src/loader/config.ts#L34

Reply:
Ah nice catch. Thank you for checking this.